kubernetes/cray-kafka-operator/values.yaml

```diff
@@ -100,3 +100,9 @@ strimzi-kafka-operator:
     requests:
       cpu: 200m
       memory: 256Mi
+
+kubectl:
+  image:
+    repository: artifactory.algol60.net/csm-docker/stable/docker-kubectl
+    tag: 1.21.12
```

Comment:
Shouldn't we be using `kubectl` of newer version, e.g. 1.22 or even 1.24 for 1.6?

Reply:
Currently in artifactory, the latest version of the kubectl image is 1.21.12 (https://artifactory.algol60.net/ui/native/csm-docker/stable/docker-kubectl/). We need to first add it antifactory for us to pick up.